File: CSharp/Array/Join.cs
using System;

public class Test
{
    public static void Main()
    {
        // 配列の宣言
        string[] datas = {"沖田", "武蔵", "モードレッド"};

        // 配列内の各要素を">"で区切って連結
        string data = string.Join(">", datas);
		
		    // 変数の中身を表示
		    System.Console.WriteLine(data); // 沖田>武蔵>モードレッド
	  }
}

File: Java/Array/Join.java
import java.util.*;
import java.lang.*;
import java.io.*;
import java.util.List;
import java.util.ArrayList;

class Test
{
    public static void main (String[] args)
    {       
        // 配列の宣言・初期化
        String[] name = {"吹雪", "白雪", "深雪"};
            
            
        String str = String.join(",", name);
            
        // コンソール出力
        System.out.println(str); // 吹雪,白雪,深雪
    }
}
